### DISCUSSION TITLE
What happens if the RabbitMQ clients are connected to follower nodes instead of leader nodes for quorum queues?

### DISCUSSION THREAD

**Discussion by veera-chenna** [rabbitmq/rabbitmq-server#10266] [Questions] [answered]
Hi All,

Till now am using a single node cluster. And now started working towards the multi node cluster. In the process have this question of how quorum queue follower nodes will handle publishers or consumers. Am looking for a documentation link for the same.

Thanks in advance

**Comment by mkuratczyk** [comment, marked as answer]
In terms of functionality, it's completely transparent. In terms of performance - there's a bit of cost since publishing happens on the leader so the node that handles the client connection needs to talk to a different node (RPC) which handles the queue leader. Consumption can happen directly from the follower.

https://blog.rabbitmq.com/posts/2020/06/quorum-queues-local-delivery/

**Comment by michaelklishin** [comment]
[What Clustering Means for Clients](https://www.rabbitmq.com/clustering.html#clustering-and-clients).
